camera.h

```diff
@@ -16,13 +16,13 @@ class Camera {
 	void RotateAroundUp(float angle);
 	void RotateAroundSide(float angle);
 
-	glm::mat4 ViewMatrix(void);
+	glm::mat4 ViewMatrix(void) const;
 
-	glm::vec3 m_Pos;
-	glm::vec3 m_Forward;
-	glm::vec3 m_Side;
-	glm::vec3 m_Up;
-	glm::quat m_Orientation;
+	glm::vec3 m_Pos{};
+	glm::vec3 m_Forward{};
+	glm::vec3 m_Side{};
+	glm::vec3 m_Up{};
+	glm::quat m_Orientation{};
```

Comment:
i am not too deep into cpps initialization problems... does this change make sense?

Reply:
Good catch. {} Initialisation is list initialisation. Just {} defaults to 0 (usually). 0 - Initialisation is cleaner than undefined values. So I'd say with {} is better.